third_party/terraform/website/docs/r/google_organization_iam_audit_config.html.markdown

```diff
@@ -47,6 +47,12 @@ The `audit_log_config` block supports:
   * **group:{emailid}**: An email address that represents a Google group. For example, admins@example.com.
   * **domain:{domain}**: A G Suite domain (primary, instead of alias) name that represents all the users of that domain. For example, google.com or example.com.
 
+## Attributes Reference
+
+In addition to the arguments listed above, the following computed attributes are exported:
+
+* `id` - an identifier for the resource with format `{{organization}}/audit_config/{{service}}`
```

Comment:
What does `organization` mean in this context? Normally it could either mean `org_id` such as `1234567` or it could be in the format `organizations/org_id` like `organizations/1234567`.

Reply:
`organization` here is the same as `org_id`. I see `xxxx_id` used here in some resources but only in very small portion. I was trying to match the format of majority cases